nuxeo-arender-rendition-apb/templates/arender_hazelcast_config.yml.j2

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: {{ name }}
+    apb_name: '{{ apb_name }}'
+    apb_plan_id: '{{ _apb_plan_id }}'
+    apb_service_class_id: '{{ _apb_service_class_id }}'
+    apb_service_instance_id: '{{ _apb_service_instance_id }}'
+  name: {{ arender_hazelcast_configmap_name }}
+  namespace: {{ namespace }}
+data:
+  hazelcast.yaml: |
+    hazelcast:
+      network:
```

Comment:
I would add a `clustername` property to be sure there is no conflict between HZ instances when we launch several clusters on the same namespace like:

```
      clustername: {{ arender_hazelcast_clustername}}
```